src/sentry/static/sentry/app/views/groupDetails/shared/groupTags.jsx

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import createReactClass from 'create-react-class';
+import {Link} from 'react-router';
+import {Box, Flex} from 'grid-emotion';
+
+import SentryTypes from 'app/sentryTypes';
+import Count from 'app/components/count';
+import DeviceName from 'app/components/deviceName';
+import LoadingError from 'app/components/loadingError';
+import LoadingIndicator from 'app/components/loadingIndicator';
+import {percent} from 'app/utils';
+import {t, tct} from 'app/locale';
+import {Panel, PanelBody, PanelHeader} from 'app/components/panels';
+import Alert from 'app/components/alert';
+import withApi from 'app/utils/withApi';
+import withOrganization from 'app/utils/withOrganization';
+
+const GroupTags = createReactClass({
+  displayName: 'GroupTags',
+
+  propTypes: {
+    organization: SentryTypes.Organization.isRequired,
+    group: SentryTypes.Group.isRequired,
+    api: PropTypes.object.isRequired,
+    query: PropTypes.object.isRequired,
+    params: PropTypes.object.isRequired,
+  },
+
+  getInitialState() {
+    return {
+      tagList: null,
+      loading: true,
+      error: false,
+    };
+  },
+
+  componentWillMount() {
+    this.fetchData();
+  },
+
+  componentWillReceiveProps(nextProps) {
```

Comment:
This could be replaced with `componentDidUpdate` as willReceiveProps is deprecated. :golf: